web/nginx/nginx.conf

```diff
@@ -25,8 +25,8 @@ http {
     ##
     # Logging Settings
     ##
-    access_log /dev/stdout;
-    error_log /dev/stdout info;
+    access_log /var/log/nginx/access.log;
+    error_log /var/log/nginx/access.log info;
```

Comment:
this seems to be the wrong file name, assuming `/var/log/nginx/error.log` is not added by mistake.

Reply:
:headdesk:

Fixed.